flake.lock

```diff
@@ -1,57 +1,122 @@
 {
```

Comment:
ideally a readme on how to use this would be amazing.

Reply:
added  a README